client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -416,7 +416,7 @@ export const SeoForm = React.createClass( {
 		const activateVerificationServices = () => this.props.activateModule( siteId, 'verification-tools' );
 		const isSitePrivate = siteSettings && parseInt( siteSettings.blog_public, 10 ) !== 1;
 		const isJetpackUnsupported = siteIsJetpack && ! jetpackVersionSupportsSeo;
-		const isDisabled = isSitePrivate || isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
+		const isDisabled = isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
```

Comment:
> In that case we should also update the notice text to ... while your site is Private or Hidden.

What if we got fancy and changed the text based on what their actual current setting is? It'd be nice to try and keep that sentence to one line when in the desktop view.